test/Microsoft.Azure.WebJobs.Logging.FunctionalTests/LoggerTest.cs

```diff
@@ -9,51 +9,60 @@
 using Microsoft.WindowsAzure.Storage;
 using Microsoft.WindowsAzure.Storage.Table;
 using Xunit;
+using System.Collections.Generic;
 
 namespace Microsoft.Azure.WebJobs.Logging.FunctionalTests
 {
-    public class LoggerTest
+    public class LoggerTest : IDisposable, IEpochTableProvider
     {
         static string CommonFuncName1 = "gamma";
 
+        private List<CloudTable> _tables = new List<CloudTable>();
+
+        private string _tableNamePrefix = "logtestYY" + Guid.NewGuid().ToString("n");
+
+        private CloudTableClient _tableClient;
+
+        // Delete any tables we created. 
+        public void Dispose()
```

Comment:
Dispose() gets called at the end of every test. I should have written the tests using this before hand rather than the try/finally. 
The changes in the test file here are primarily cosmeitc - there's no 'functional" change.